### PR TITLE
Fix harness.yml to use /Backoffice as root_dir for spryker versions newer 202108.0

### PR DESCRIPTION
**Pull request by sebastianlarisch**
With Release 202108.0 Spryker has implemented separate application bootstrapping into individual endpoints.
The harness.yml was already updated to use /BackendGateway as the endpoint for the ZED API for Spryker demoshop version >= 202108.0, but the Zed endpoint is still pointing to /Zed. Following Sprykers convention it should be /Backoffice instead.

### CODE REVIEW

**Review by hgajjar — APPROVED** [review]
👍

File: src/spryker/harness.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ attributes:
       DE: = 'zed-de-' ~ @('hostname')
       AT: = 'zed-at-' ~ @('hostname')
       US: = 'zed-us-' ~ @('hostname')
-    root_dir: = @('app.web_directory') ~ '/Zed'
+    root_dir: "= @('spryker.demoshop-version') >= '202108.0'? @('app.web_directory') ~ '/Backoffice' : @('app.web_directory') ~ '/Zed'"
```
Comment (hgajjar) [inline src/spryker/harness.yml]: Actually, it's better to use the `version_compare` function here. Example: https://github.com/inviqa/harness-base-php/blob/1.5.x/src/spryker/harness.yml#L92

**Review by hgajjar — CHANGES_REQUESTED** [review]

File: src/spryker/harness.yml (hunk 1 above)
Comment (sebastianlarisch) [reply src/spryker/harness.yml]: k, changed it

**Comment by Chemaclass** [comment]
Any update regarding this PR? 🎻

**Pull request closed without merging**